lua/gh-actions/github.lua

```diff
@@ -57,24 +62,30 @@ end
 
 ---@param cmd? string
 ---@return string
-function M.get_github_token(cmd)
+function M.get_github_token(cmd, server)
```

Comment:
Missing type annotation for `server` param. I'd also switch the order of the arguments, as `server` is now always passed, while `cmd` is still optional.

Reply:
Uh, thanks for catching that! I'm not sure if it makes sense to switch the parameters though, as it would make server mandatory. Currently in the tests just cmd is passed, which is still fine this way and will default to github.com. But if you think it would be better to require specifying `server`, I can do that :) . What do you think?

Reply:
I see, I'd say then the test should also change and accommodate for that change. But the test suite isn't really thorough anyway :see_no_evil: 

Lets merge it like it is now, thank you!